### PR TITLE
Updated delete check and added new test case

### PR DESCRIPTION
**Pull request by donaldchen34**
Description of PR here...

Issue: Closes #

### Frontend:
- 
### Backend:
- added new check in the doAction func to check if location is given
-  added new testcase in Test.py

### CODE REVIEW

File: backend/test.py [hunk 1]
```diff
@@ -158,5 +158,23 @@ def environment_test():
     print(results)
     return results
 
-environment_test()
-chatbot_test()
\ No newline at end of file
+#Delete with a capital D is recognized as a noun and will not work
```
Comment (joshcwinton) [inline backend/test.py:165]: can we convert everything to lowercase once we get the input to avoid this?

File: backend/chatbot.py [hunk 2]
```diff
@@ -72,15 +72,27 @@ def doAction(action,shape,color,row,col):
         else:
             return 1 #Say location is taken
 
-    foundShape = findShape(shape=shape,color=color)
-    #Ask check for location
+    foundShape = []
+    if row != -1 and col != -1 and not emptyPosition(row=row, col=col):
```
Comment (joshcwinton) [inline backend/chatbot.py]: are you able to do `emptyPosition(row, col)`

**Review by joshcwinton — CHANGES_REQUESTED** [review]

File: backend/test.py (hunk 1 above)
Comment (donaldchen34) [reply backend/test.py:165]: done

Comment (joshcwinton) [reply backend/test.py:165]: 🤝🤝🤝🤝

**Review by joshcwinton — APPROVED** [review]
Excellent work 😈

**Pull request merged**